extensions/README.md

```diff
@@ -60,6 +60,7 @@ An extension can add new fields to STAC entities (content extension), or can add
 | [Scientific](scientific/README.md) (`sci`)                   | Item, Collection | Scientific metadata is considered to be data that indicate from which publication data originates and how the data itself should be cited or referenced. | *Proposal* |
 | [Single File STAC](single-file-stac/README.md) (-)           | ItemCollection   | An extension to provide a set of Collections and Items as a single file catalog. | *Proposal* |
 | [Versioning Indicators](version/README.md) (-)               | Item, Collection | Provides fields and link relation types to provide a version and indicate deprecation. | *Proposal* |
+| [Tiled Assets](tiled-assets/README.md) (`tl`)                | Item             | Allows to specify numerous assets using asset templates via tile matrices and dimensions. | *Proposal* |
```

Comment:
It's still tl here. Also there are merge conflicts for this file.

Reply:
Sorry, missed that.
I merged dev and resolved the conflicts now.